### ISSUE TITLE
Execute default actions from custom action Button?

### ISSUE THREAD

**Issue by joshbooker** [SharePoint/sp-dev-docs#6651]
## Category

- [ X] Question
- [ ] Typo
- [ ] Additional article idea

## Question

I understand default menu and command bar actions cannot be changed, but can we call those same actions from a custom button?  Is it possible to execute a built-in action from a custom action button?  

For example, how would I execute the built-in Approve\Reject action from my own button?  Say with a [ListView command set extension](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/get-started/building-simple-cmdset-with-dialog-api)?

Basically I want to move the Approve/Reject command out from beneath the elipses on the command bar.

TIA,
Josh

**Comment by andrewconnell** [comment]
At this time no, there isn't a vehicle to call existing logic on existing buttons.

**Comment by joshbooker** [comment]
Thanks for your reply.  Do you know of any other way to execute the built-in Approve\Reject action?  there used to be approve.aspx but I believe that's not modern experience.
